fastexcel-writer/src/main/java/org/dhatim/fastexcel/DataValidation.java

```diff
@@ -0,0 +1,27 @@
+package org.dhatim.fastexcel;
+
+import java.io.IOException;
+
+/**
+ *
+ */
+public interface DataValidation {
+
+    /**
+     * Write this dataValidation as an XML element.
+     *
+     * @param w Output writer.
+     * @throws IOException If an I/O error occurs.
+     */
+    void write(Writer w) throws IOException;
+
+    /**
+     * Construct a new ListDataValidation
+     *
+     * @param range     The Range this validation is applied to
+     * @param listRange The Range of the list this validation references
+     */
+    static ListDataValidation list(Range range, Range listRange) {
```

Comment:
Maybe it would be nicer to build a validation from the range we want to validate. So we could get rid of this factory and we would not need to expose `DataValidation` interface and implementations.
This could result in the more expressive example:
```java
ws.range(<to validate>).validateWithList(ws.range(<coordinates of list>));
```
What do you think?

Reply:
I like this, I'll make the changes